packages/extension/app/background.js

```diff
@@ -61,6 +61,16 @@ const connect = function (host, path, extraOpts) {
     // adds a header to the request to mark it as a request for the AUT frame
     // itself, so the proxy can utilize that for injection purposes
     browser.webRequest.onBeforeSendHeaders.addListener((details) => {
+      const defaultHeaders = {
```

Comment:
I think this should be called `requestModifications` or something. This object currently only includes headers, but [it can include various other properties](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/BlockingResponse). MDN calls it `webRequest.BlockingResponse` but that's a little confusing since this modifying a request.

Reply:
I think that is a better name and reflects more accurately what is going on. I will update here and in electron!

Reply:
addressed in a2ad929192